ibm/service/kms/resource_ibm_kms_key_test.go

```diff
@@ -30,13 +30,30 @@ func TestAccIBMKMSResource_basic(t *testing.T) {
 		Providers: acc.TestAccProviders,
 		Steps: []resource.TestStep{
 			{
-				Config: testAccCheckIBMKmsResourceConfig(instanceName, resourceName, keyName, standard_key),
+				// Test Imported Standard Key
+				Config: testAccCheckIBMKmsResourceImportConfig(instanceName, resourceName, keyName, standard_key, payload),
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr("ibm_kms_key.test", "key_name", keyName),
 				),
 			},
 			{
-				Config: testAccCheckIBMKmsResourceImportStandardConfig(instanceName, resourceName, keyName, payload),
+				// Test Imported Root Key
+				Config: testAccCheckIBMKmsResourceImportConfig(instanceName, resourceName, keyName, !standard_key, payload),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("ibm_kms_key.test", "key_name", keyName),
+				),
+				ExpectNonEmptyPlan: true,
```

Comment:
Why this condition added?

Reply:
It was failing by saying the plan was not empty.